device-types/Lenovo/SR850.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Lenovo
+model: ThinkSystem SR850
+slug: sr850
+part_number: 7B27A03953
```

Comment:
This part number appears to only be for the processor and memory expansion tray. This is not the part number for the device itself.